cli/lib/cli.js

```diff
@@ -395,18 +391,12 @@ module.exports = {
     .option('--dev', text('dev'), coerceFalse)
     .action((opts) => {
       debug('opening Cypress')
-      require('./exec/open')
-      .start(util.parseOpts(opts))
-      .catch(util.logErrorExit1)
+      require('./exec/open').start(util.parseOpts(opts)).catch(util.logErrorExit1)
     })
 
-    addCypressRunCommand(program)
-    .action((...fnArgs) => {
+    addCypressRunCommand(program).action((...fnArgs) => {
       debug('running Cypress with args %o', fnArgs)
-      require('./exec/run')
-      .start(parseVariableOpts(fnArgs, args))
-      .then(util.exit)
-      .catch(util.logErrorExit1)
+      require('./exec/run').start(parseVariableOpts(fnArgs, args)).then(util.exit).catch(util.logErrorExit1)
```

Comment:
I feel like all  of these calls in the CLI have ended up a lot less clear to read fwiw.

Reply:
I agree. This is because of the changes Brian requested.